impeller/display_list/display_list_unittests.cc

```diff
@@ -676,7 +698,7 @@ TEST_P(DisplayListTest, CanDrawShadow) {
   }
 
   ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
-}
+}  // namespace testing
```

Comment:
Nit: Don't think this is the end of the namespace. Perhaps your editor is confused?

Reply:
Whoops! I trigger clang format runs in the middle of writing code often. Fixed.